hack/update-codegen.sh

```diff
@@ -18,7 +18,7 @@ function cleanup {
 }
 trap cleanup EXIT
 
-go install ./${CODEGEN_PKG}/cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
+go install --mod=vendor ./${CODEGEN_PKG}/cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
```

Comment:
Without `--mod=vendor` I was getting the following failure when using go 1.15
```
without -mod=vendor, directory /home/umohnani/go/src/github.com/openshift/client-go/vendor/k8s.io/code-generator/cmd/defaulter-gen has no package path
without -mod=vendor, directory /home/umohnani/go/src/github.com/openshift/client-go/vendor/k8s.io/code-generator/cmd/client-gen has no package path
without -mod=vendor, directory /home/umohnani/go/src/github.com/openshift/client-go/vendor/k8s.io/code-generator/cmd/lister-gen has no package path
without -mod=vendor, directory /home/umohnani/go/src/github.com/openshift/client-go/vendor/k8s.io/code-generator/cmd/informer-gen has no package path
without -mod=vendor, directory /home/umohnani/go/src/github.com/openshift/client-go/vendor/k8s.io/code-generator/cmd/deepcopy-gen has no package path
```

Reply:
Yeah, we advice folks to have `GOFLAGS=-mod-vendor` set in their env since we use vendor and not the default go.mod mechanism. But you shouldn't set this here explicitly. We do that already in the build scripts in CI.

Reply:
Alright, added it to the readme instead.